### ISSUE TITLE
value_counts operation is too slow, nearly stuck

### ISSUE THREAD

**Issue by universewill** [modin-project/modin#763]
i found that the 'value_count' operation will stuck:
```
UserWarning: `Series.value_counts` defaulting to pandas implementation.
To request implementation, send an email to feature_requests@modin.org.
UserWarning: Distributing <class 'pandas.core.series.Series'> object. This may take some time.
```
It seems that modin tries to convert modin df to pandas df, and it is very slow.

**Comment by devin-petersohn** [comment]
Hi @universewill, thanks for posting.

This is not yet implemented in a distributed way, I will tag as Feature Request and Help Wanted. Let me know if you'd like to contribute!

**Comment by anmyachev** [comment]
Distributed `value_counts` was be implemented in #1535. Now this operation should be faster.

@universewill feel free to reopen this issue if there will be performance issue.
